README.md

```diff
@@ -76,21 +76,22 @@ is recommended to export XDG variables in general, but if you have not, just
 replace the usages of `$XDG_DATA_HOME` below with `$HOME/.local/share`, which is
 the usual default for this XDG variable.
 
-    mkdir -p $XDG_DATA_HOME/llvm/firefly
```

Comment:
This change, and the introduction of the `mv` line below are incorrect - there is a difference here between Linux and macOS due to differing `tar` implementations, and on Linux we extract directly into the target directory. Can you revert the changes to this section? Everything else looks good

Reply:
Thank you for spotting my mistakes. I reverted the changes in Linux commands.